src/seedu/addressbook/AddressBook.java

```diff
@@ -568,6 +600,39 @@ private static String executeClearAddressBook() {
         return MESSAGE_ADDRESSBOOK_CLEARED;
     }
 
+    /**
+     * Updates person identified using last displayed index.
+     *
+     * @param commandArgs full command args string from the user
+     * @return feedback display message for the operation result
+     */
+    private static String executeUpdatePerson(String commandArgs)   {
+        final String[] updateStringSplit = splitIndexAndPersonData(commandArgs);
+
+        if (!isDeletePersonArgsValid(updateStringSplit[0])) {
+            return getMessageForInvalidCommandInput(COMMAND_UPDATE_WORD, getUsageInfoForUpdateCommand());
+        }
+        final int targetVisibleIndex = extractTargetIndexFromDeletePersonArgs(updateStringSplit[0]);
```

Comment:
Again, renaming the method would make it easier to understand what is going on here.